back/app/services/sanitization_service.rb

```diff
@@ -149,25 +149,25 @@ class IframeScrubber < Rails::Html::PermitScrubber
 
     attr_reader :tags, :attributes
 
-    def initialize(features)
+    def initialize(features = [])
       super()
       features_w_default = features.concat([:default])
       @tags = features_w_default.flat_map { |f| EDITOR_FEATURES[f][:tags] }.uniq
       @attributes = features_w_default.flat_map { |f| EDITOR_FEATURES[f][:attributes] }.uniq
     end
 
     def allowed_node?(node)
-      return iframe_allowed? && video_whitelisted?(node) if node.name == 'iframe'
+      return iframe_allowed? && video_whitelisted?(node['src']) if node.name == 'iframe'
       ensure_nofollow(node) if node.name == 'a'
       tags.include? node.name
     end
 
-    private
-
-    def video_whitelisted?(node)
-      VIDEO_WHITELIST.any? { |regex| (node['src'] =~ regex)&.zero? }
+    def video_whitelisted?(url)
+      VIDEO_WHITELIST.any? { |regex| (url =~ regex)&.zero? }
```

Comment:
Consider using `match?` instead of `=~` and checking for `&zero`. `match?` expresses intent better.